src/export/MappingExporter.ts

```diff
@@ -72,7 +72,38 @@ export class MappingExporter {
       Type.Extension
     ) as StructureDefinition;
     if (sourceStructDef) {
-      this.setMetadata(sourceStructDef, fshDefinition);
+      const parent = this.fisher.fishForFHIR(
+        sourceStructDef.baseDefinition,
+        Type.Resource,
+        Type.Type,
+        Type.Profile,
+        Type.Extension
+      );
+      const matchingParentMapping = parent?.mapping.find(
+        (m: StructureDefinitionMapping) => m.identity === fshDefinition.id
+      );
+      if (matchingParentMapping != null) {
+        const isMatchingTitle = fshDefinition.title
+          ? fshDefinition.title === matchingParentMapping.name
+          : true;
+        const isMatchingTarget = fshDefinition.target
+          ? fshDefinition.target === matchingParentMapping.uri
+          : true;
+        const isMatchingDescription = fshDefinition.description
+          ? fshDefinition.description === matchingParentMapping.comment
+          : true;
+        if (!isMatchingTitle || !isMatchingTarget || !isMatchingDescription) {
+          // If the mapping identity matches one on the parent, all other metadata must also match in order to merge MappingRules
```

Comment:
I know this is _exactly_ what I suggested when we discussed this, but now I'm having second thoughts...  If the `mapping.comment` is the only thing that changes, is that so bad?  Is it possible that in a profile you might want to provide additional comments about the mapping (in general) that maybe didn't apply to the mapping in the parent?  I think maybe that should be allowed.

So I guess I'm suggesting we still check for matching `name` (`title`) and `uri` (`target`) but perhaps we allow for changes in `comment` (`description`).  What do you think?

Reply:
BTW -- at first I thought maybe even requiring `name` (`title`) to match was too strict, but when I looked at the definition in FHIR, it says "A name for the specification that is being mapped to" and says you _must_ have either a `uri` _or_ `name` -- so I think that probably _shouldn't_ change (but `comment` still seems changeable to me).

Reply:
Second thoughts are always allowed! I think I'm fine with this change! But out of curiosity, do you have a link to that documentation you're referencing? I've been struggling to find the right FHIR documentation for Mappings.

Reply:
This is what I was looking at: http://hl7.org/fhir/R4/structuredefinition-definitions.html#StructureDefinition.mapping

Reply:
Ah okay. I've been there so I at least feel better about that! And I agree with your second comment. It feels like if you should have a `uri` or `name`, we should match both of those. But a `comment` seems reasonable for a profile to change. So I'll make that update!